app/cdap/components/Operations/JobsTable/index.tsx

```diff
@@ -67,40 +66,42 @@ const JobsTitleView: React.FC<IJobsTableProps> = ({ classes, data }) => {
         </TableHeader>
 
         <TableBody>
-          {data.jobs.map((row) => {
-            const endTime = row.end ? row.end : Math.floor(Date.now() / 1000);
-            const duration = endTime - row.start;
-            const namespace = row.namespace;
-            const appName = row.application.name;
+          {data.jobs
+            .filter((row) => (isCurrentTimeFrame ? true : row.status !== PROGRAM_STATUSES.RUNNING))
```

Comment:
The filter condition can be simplified to `isCurrentTimeFrame || row.status !== PROGRAM_STATUSES.RUNNING`